aliases/check_consistency.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: "check_consistency"
+pack: "napalm"
+action_ref: "napalm.check_consistency"
+description: "Check consistency of the device's configuration"
+formats:
+  - "check consistency on {{hostname}}"
```

Comment:
@Mierdin Since there would be other packs as well on the ST2 installation, does it make sense to give the alias a keyword at the beginning so that chatops commands for this pack are easily identifiable? Some like `napalm check consistency on {{hostname}}`. I normally do this for integrations.